pkg/ucauth/user_admin.go

```diff
@@ -342,7 +342,11 @@ func (c *UCClient) findUsersByQuery(query string, idOrder ...string) ([]User, er
 		}
 		var orderedUsers []User
 		for _, id := range idOrder {
-			orderedUsers = append(orderedUsers, userMap[id])
+			user, ok := userMap[id]
```

Comment:
how to proven there exist some case `ok = false`?

Reply:
For example, there are two user IDs, one exists in uc and the other does not exist. Using these two user IDs to query uc, only existing users will be returned. So the not-existed userID will `ok = false`

Reply:
In which situation "the caller pass non-exist userID", as you say only the caller give use bad userID will cause this problem, and what if there are no bad-caller?